docs/source/python_api/mlflow.models.rst

```diff
@@ -8,3 +8,5 @@ mlflow.models
 
 .. autoclass:: mlflow.models.model.ModelInfo
     :members:
+
+.. autofunction:: mlflow.models.model.update_model_requirements
```

Comment:
Is `update_model_requirements` imported in `mlflow.models`?

Reply:
It's not

Reply:
can you import it?

Reply:
Never mind if `mlflow.models.model` is already public.